RLBotDiscordBot/bot.py

```diff
@@ -36,17 +38,35 @@ def __init__(self):
         self.remove_command('help')
 
         for extension in initial_extensions:
+            #print("woof")
             try:
                 self.load_extension(extension)
             except Exception as e:
                 self.logger.error(f'There was an error loading the extension {extension}. Error: {e}')
 
     async def on_ready(self):
         self.logger.info(f'{self.user} online! (ID: {self.user.id})')
+        self.has_reacted = 0
+        self.has_checked = False
 
     async def on_message(self, message):
+        if not self.has_checked:
+            x = message.guild.members
+            z = message.guild.roles
+            for member in x:
+                y = member.roles
+                for role in y:
+                    if role == get(member.guild.roles, name="Python") or role == get(member.guild.roles, name="Java") or role == get(member.guild.roles, name="C#") or role == get(member.guild.roles, name="Rust") or role == get(member.guild.roles, name="Scratch"):
```

Comment:
Can't you just do `role.name == "Python"`? https://github.com/Rapptz/discord.py/blob/12343c142262c675ca4b0483ad9bb099c9fac835/discord/role.py#L74

Reply:
Probably, it was just being a bit dodgy when I was testing it